browser/resources/brave_new_tab_page_refresh/components/widgets/rewards_widget.tsx

```diff
@@ -169,6 +182,67 @@ export function RewardsWidget() {
     )
   }
 
+  function renderPayoutStatus() {
+    if (earnings <= 0 || !externalWallet) {
+      return null
+    }
+    const status =
+      getProviderPayoutStatus(payoutStatus, externalWallet.provider)
+    if (status === 'complete') {
+      return (
+        <div className='payout-status'>
+          {
+            formatMessage(getString('rewardsPayoutCompletedText'), [
+              getPayoutMonth()
+            ])
+          }
+        </div>
+      )
+    }
+    if (status === 'processing') {
+      return (
+        <div className='payout-status'>
+          {
+            formatMessage(getString('rewardsPayoutProcessingText'), [
+              getPayoutMonth()
+            ])
+          }
+          <Link url={urls.payoutStatusURL}>
+            {getString('rewardsPayoutDetailsLink')}
+          </Link>
+        </div>
+      )
+    }
+    return null
+  }
+
+  function renderAdsViewed() {
+    if (adsViewed === null) {
+      return null
+    }
+    return (
+      <div className='ads-viewed'>
+        {
+          formatMessage(adsViewedString, {
+            tags: {
+              $1: (content) => (
+                <span key='ad-count' className='ad-count'>
+                  {content}
+                </span>
+              )
+            }
+          })
```

Comment:
Should we use `formatLocale` here instead, as we deprecated this? It means we'll have less to migrate later on.

Reply:
Unfortunately some of the rewards strings use the older placeholder protocol and are shared with the Rewards page. https://github.com/brave/brave-browser/issues/48506